CHANGES.md

```diff
@@ -155,6 +155,8 @@ To be released.
     specific peer. [[#1240]]
  -  Added the option `--no-passphrase` to `planet key remove` command to remove
     key without asking passphrase.  [[#1213], [#1265]]
+ -  Added `planet key derive PRIVATE-KEY` subcommand to derive the address or
+    public key from a private.  [[#1268]]
```

Comment:
For consistency… (We've never shown metavariables for CLI commands/subcommands in this document.)

```suggestion
 -  Added `planet key derive` subcommand to derive the address or
    public key from a private.  [[#1268]]
```